packages/data/src/types.ts

```diff
@@ -182,7 +182,7 @@ export type ActionCreatorsOf< Config extends AnyConfig > =
 // return type of each action creator to account for internal registry details --
 // for example, dispatched actions are wrapped with a Promise.
 export type PromisifiedActionCreators<
-	ActionCreators extends MapOf< ActionCreator >
+	ActionCreators extends MapOf< ActionCreator >,
```

Comment:
The comma after a single generic parameter definitely looks odd 🤔

Reply:
And yet it's perfectly logical, that's how `trailingComma` should work. The idea is when you add a second type parameter, it's a one-line diff, not two-line.

Reply:
That rationale makes sense to me 👍